prisma/schema.prisma

```diff
@@ -6,32 +6,33 @@ generator client {
 }
 
 datasource db {
-  provider = "postgresql"
-  url      = env("POSTGRES_PRISMA_URL")
+  provider  = "postgresql"
+  url       = env("POSTGRES_PRISMA_URL")
   directUrl = env("POSTGRES_URL_NON_POOLING")
 }
 
 model Setting {
-  id                String      @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
-  bannerImageId     String?     @db.Uuid @unique
-  bannerImage       Media?      @relation(fields: [bannerImageId], references: [id], onDelete: SetNull)
-  backgroundColor   String?
-  content           String?
-  createdById       String      @db.Uuid
-  createdAt         DateTime    @default(now()) @db.Timestamptz()
-  updatedAt         DateTime    @updatedAt @ignore @db.Timestamptz()
+  id              String   @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  bannerImageId   String?  @unique @db.Uuid
+  bannerImage     Media?   @relation(fields: [bannerImageId], references: [id], onDelete: SetNull)
+  backgroundColor String?
+  content         String?
+  workspaceId     String
+  createdById     String   @db.Uuid
+  createdAt       DateTime @default(now()) @db.Timestamptz()
+  updatedAt       DateTime @updatedAt @ignore @db.Timestamptz()
 
   @@index([bannerImageId], name: "bannerImageId")
 }
 
 model Media {
-  id                String      @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
-  url               String
-  filename          String      
-  contentType       String
-  size              Int
-  createdById       String      @db.Uuid
-  createdAt         DateTime    @default(now()) @db.Timestamptz()
-  updatedAt         DateTime    @updatedAt @ignore @db.Timestamptz() 
-  setting           Setting? 
+  id          String   @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
```

Comment:
I think we have formatting issue here. May be eslint/prettier setting not working